src/main/java/Duke.java

```diff
@@ -1,10 +1,17 @@
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Scanner;
 
-
 public class Duke {
     private static ArrayList<Task> tasks = new ArrayList<>();
 
+    private static String isPlural(int n) {
```

Comment:
I think it may be better if this method is renamed, since it may be confused with one that returns a boolean true or false value.